packages/link/src/Link.Neutral.story.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withInfo } from "@storybook/addon-info";
+
+import Link from "./Link";
+import linkTypes from "./constants/linkTypes";
+
+storiesOf("Link/Neutral", module)
+  .add(
+    "Default - left aligned",
+    withInfo()(() => (
+      <Link href="#" type={linkTypes.NEUTRAL}>
+        Awesome Link
+      </Link>
+    ))
+  )
+  .add(
+    "right aligned",
+    withInfo()(() => (
+      <div style={{ width: "500px", textAlign: "right" }}>
```

Comment:
when a value is in pixels, React allows it to be set as number, just a tip, not a request